### ISSUE TITLE
Docker throws errors when shutting down devstack

### ISSUE THREAD

**Issue by aronchick** [filecoin-project/bacalhau#594]
Probably low pri - but 

```
^C15:24:03.21 | INF bacalhau/devstack.go:151 > Shutting down devstack
15:24:03.417 | ERR system/cleanup.go:85 > Error during clean-up callback: close tcp4 0.0.0.0:60267: use of closed network connection
15:24:03.418 | ERR system/cleanup.go:85 > Error during clean-up callback: close tcp4 0.0.0.0:60290: use of closed network connection
15:24:03.418 | ERR system/cleanup.go:85 > Error during clean-up callback: close tcp4 0.0.0.0:60278: use of closed network connection
15:24:03.418 | ERR docker/executor.go:387 > Docker executor stop error: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1&limit=0": context canceled
15:24:03.418 | ERR docker/executor.go:387 > Docker executor stop error: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1&limit=0": context canceled
15:24:03.418 | ERR docker/executor.go:387 > Docker executor stop error: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1&limit=0": context canceled
```

Should Docker get its own context?

**Comment by aronchick** [comment]
if you refactor devstack - @binocarlos

**Comment by wjam** [comment]
Duplicate of #893 and fixed by #894
